BlockchainSdk/Extensions/Task+.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Task+.swift
+//  BlockchainSdk
+//
+//  Created by Dmitry Fedorov on 08.04.2024.
+//
+
+import Foundation
+
+extension Task where Failure == Error {
+    @discardableResult
+    static func retrying(
+        priority: TaskPriority? = nil,
+        maxRetryCount: Int = 1,
+        retryDelay: TimeInterval = 0,
+        operation: @Sendable @escaping () async throws -> Success
+    ) -> Task {
+        Task(priority: priority) {
+            for _ in 0..<maxRetryCount {
+                do {
+                    return try await operation()
+                } catch {
+                    let oneSecond = TimeInterval(1_000_000_000)
+                    let delay = UInt64(oneSecond * retryDelay)
```

Comment:
можно вот так множитель взять 
```suggestion
                    let delay = UInt64(N_SEC_PER_SEC * retryDelay)
```